web/app/services/fetch.ts

```diff
@@ -33,13 +42,9 @@ export default class FetchService extends Service {
       }
 
       if (!resp.ok) {
-        if (resp.status === 401) {
-          if (isPollCall) {
-            // handle poll-call failures via the session service
-            return;
-          } else {
-            this.session.invalidate();
```

Comment:
Since we invalidate on applicationError, calling it here seems redundant at best.